src/panels/lovelace/components/hui-image.ts

```diff
@@ -209,13 +279,22 @@ export class HuiImage extends LitElement {
   }
 
   private _onImageError(): void {
-    this._loadError = true;
+    this._loadState = LoadState.Error;
   }
 
   private async _onImageLoad(): Promise<void> {
-    this._loadError = false;
+    this._loadState = LoadState.Loaded;
+    this._lastImageHeight = this._image?.offsetHeight;
     await this.updateComplete;
-    this._lastImageHeight = this._image.offsetHeight;
```

Comment:
Why was this swapped?

Reply:
Not sure at this point. Was likely due to the merge conflict.

Reply:
I don't *think* it makes a difference either way

Reply:
It seems wrong now to me, as we don't await the render of the image before we measure it

Reply:
Although, when it is loaded it will be rendered...? Not sure why it was there before then...